tests/test_evaluate.py

```diff
@@ -73,17 +73,33 @@ def test_metrics_singletask_clf(
 ):
     _, test = test_single_task_benchmark_clf.get_train_test_split()
     predictions = np.random.randint(2, size=test.inputs.shape[0])
-    result = test_single_task_benchmark_clf.evaluate(predictions)
+    probabilities = np.random.uniform(size=test.inputs.shape[0])
+    result = test_single_task_benchmark_clf.evaluate(predictions, probabilities)
     for metric in test_single_task_benchmark_clf.metrics:
         assert metric in result.results.Metric.tolist()
 
 
+def test_metrics_singletask_multicls_clf(
+    tmpdir: str, test_single_task_benchmark_multi_clf: SingleTaskBenchmarkSpecification
+):
+    _, test = test_single_task_benchmark_multi_clf.get_train_test_split()
+    predictions = np.random.randint(3, size=test.inputs.shape[0])
+    probablities = np.random.random(size=(test.inputs.shape[0], 3))
+    probablities = probablities / probablities.sum(axis=1, keepdims=True)
+    result = test_single_task_benchmark_multi_clf.evaluate(predictions, probablities)
+    for metric in test_single_task_benchmark_multi_clf.metrics:
+        assert metric in result.results.Metric.tolist()
+
+
 def test_metrics_multitask_clf(tmpdir: str, test_multi_task_benchmark_clf: MultiTaskBenchmarkSpecification):
     train, test = test_multi_task_benchmark_clf.get_train_test_split()
     predictions = {
         target_col: np.random.randint(2, size=test.inputs.shape[0]) for target_col in train.target_cols
     }
```

Comment:
This can be removed, right?

Reply:
The evaluation will take both 'predictions' and 'probabilities' as input for all the defined metrics.